tests/integration-tests/tests/update/test_update.py

```diff
@@ -464,9 +484,12 @@ def test_update_awsbatch(region, pcluster_config_reader, clusters_factory, test_
     # Verify initial configuration
     _verify_initialization(region, cluster, cluster.config)
 
-    # Update cluster with the same configuration, command should not result any error even if not using force update
-    cluster.config_file = str(init_config_file)
-    cluster.update(force=False)
+    # Update cluster with the same configuration. Command should detect there is no change
+    _assert_no_update(cluster.name, init_config_file)
+
+    # Update cluster with another cluster section, where the configurations are the same as default cluster section.
+    # Command should treat the two sections as the same and detect there is no change
+    _assert_no_update(cluster.name, init_config_file, cluster_section_label="default2")
```

Comment:
I'd do this check in the unit tests; this integration test could probably be left as it was. Let's talk about it if you want.

Reply:
So are you saying we do not need any integration test for this task?